source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/sc_lambda_lcao.h

```diff
@@ -25,15 +25,20 @@ class OperatorScLambda<OperatorLCAO<TK, TR>> : public OperatorLCAO<TK, TR>
     OperatorScLambda<OperatorLCAO<TK, TR>>(LCAO_Matrix* LM_in,
                                     const std::vector<ModuleBase::Vector3<double>>& kvec_d_in,
                                     hamilt::HContainer<TR>* hR_in,
-                                    std::vector<TK>* hK_in)
-        : OperatorLCAO<TK, TR>(LM_in, kvec_d_in, hR_in, hK_in)
+                                    std::vector<TK>* hK_in,
+                                    const std::vector<int>& isk_in)
+        : isk(isk_in), OperatorLCAO<TK, TR>(LM_in, kvec_d_in, hR_in, hK_in)
     {
         this->cal_type = lcao_sc_lambda;
     }
 
     virtual void contributeHR() override;
 
     virtual void contributeHk(int ik) override;
+  private:
+
+    bool HR_fixed_done = false;
```

Comment:
Why do you add this variable?

Reply:
This variable is useless. I will delete it then.